src/entities/trade.ts

```diff
@@ -198,13 +198,21 @@ export class Trade<TInput extends Currency, TOutput extends Currency, TTradeType
       return this._priceImpact
     }
 
+    if (this.outputTax.equalTo(ONE_HUNDRED_PERCENT)) {
+      throw new Error('Unable to calculate price impact for a 100% buy-tax token')
```

Comment:
should we be throwing here? imo probably not because it makes the devx pretty rough for people using this SDK